manager-bundle/src/ContaoManager/ApiCommand/RemoveDotEnvCommand.php

```diff
@@ -49,14 +50,18 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $key = $input->getArgument('key');
         $file = $this->projectDir.'/.env';
+        $fileLocal = $file.'.local';
 
-        if (!file_exists($file)) {
-            return 0;
+        $dotenv = new DotenvDumper($fileLocal);
+
+        if (file_exists($file) && isset((new Dotenv(false))->parse(file_get_contents($file))[$key])) {
```

Comment:
Shouldn‘t this be `$fileLocal` here?

Reply:
No, if the `.env` file exists and the `$key` is set in that file, we have to set the `$key` to the empty value in `.env.local` in order to “remove” that value.

If `$key` does not exist in `.env` we can simply remove it from `.env.local`.

Reply:
See `RemoveDotEnvCommandTest::testSetsToEmptyStringIfNonLocalFileExists()`